sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -110,6 +112,15 @@ def __init__(self,
         data_channel_factory=self._data_channel_factory,
         fns=self._fns)
 
+    if status_address:
```

Comment:
I need to do the actual initialization inside sdk_worker since I want to pass the active bundle cache in sdk worker in order to report the dangling operation.

Reply:
Sounds good